ext/objspace/objspace_dump.c

```diff
@@ -268,8 +268,8 @@ dump_object(VALUE obj, struct dump_config *dc)
 
       case T_CLASS:
       case T_MODULE:
-	if (dc->cur_obj_klass)
-	    dump_append(dc, ", \"name\":\"%s\"", rb_class2name(obj));
+	if (dc->cur_obj_klass && !NIL_P(rb_mod_name(obj)))
+	    dump_append(dc, ", \"name\":\"%s\"", RSTRING_PTR(rb_mod_name(obj)));
```

Comment:
Calling `rb_mod_name` twice?

Reply:
Yeah, I figured it was cheap enough that it wasn't worth declaring a variable. But I can change that.